docs/2.Basics/Hard and soft fail.md

```diff
@@ -62,3 +62,17 @@ The table below shows how different values of `--soft-fail`, `--soft-fail-on`, a
 |False|LOW,CKV_789|CKV_123|1|The explicit match of the hard fail criteria results in a hard fail|
 |False|CKV_789|HIGH|1|CKV_789 explicitly matches a soft fail criteria, which overrides the hard fail. But CKV_123 is not in either list, so defaults to the value of `--soft-fail`, which is false|
 |True|CKV_789|HIGH|0|CKV_789 explicitly matches a soft fail criteria, which overrides the hard fail. But CKV_123 is not in either list, so defaults to `--soft-fail`, which is true|
+
+# Platform enforcement rules
+
+Checkov can download [enforcement rules](https://docs.bridgecrew.io/changelog/enforcement-rules-settings) that you configure in the Bridgecrew or Prisma Cloud platform. This allows you to centralize the failure and check threshold configurations, instead of defining them in each pipeline.
```

Comment:
Probably better to point to the docs than the changelog: https://docs.bridgecrew.io/docs/enforcement